hindsight/modules/heka/elasticsearch/payload.lua

```diff
@@ -0,0 +1,41 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
not needed this also in the sandbox
http://mozilla-services.github.io/lua_sandbox/modules/heka/elasticsearch/payload.html